pkg/ui/src/views/sessions/sessionsTable.tsx

```diff
@@ -0,0 +1,218 @@
+      title: SessionTableTitle.sessionAge,
+      className: cx("cl-table__col-session-age"),
+      cell: session => SessionLink({session: session.session}),
+      sort: session => TimestampToMoment(session.session.start).valueOf(),
+    },
+    {
+      title: SessionTableTitle.txnAge,
+      className: cx("cl-table__col-session-start"),
+      cell: function(session: SessionInfo) {
+        if (session.session.active_txn) {
+          return AgeLabel({
+            start: TimestampToMoment(session.session.active_txn.start),
+            thingName: "Transaction",
+          });
+        }
+        return "N/A";
+      },
+      sort: session => session.session.active_txn?.start.seconds || 0,
+    },
+    {
+      title: SessionTableTitle.statementAge,
+      className: cx("cl-table__col-session-start"),
+      cell: function(session: SessionInfo) {
+        if (session.session.active_queries?.length > 0) {
+          return AgeLabel({
+            start: TimestampToMoment(session.session.active_queries[0].start),
+            thingName: "Query",
+          });
+        }
+        return "N/A";
+      },
+      sort: function(session: SessionInfo): number{
+        if (session.session.active_queries?.length > 0) {
+          return session.session.active_queries[0].start.seconds.toNumber();
+        }
+        return 0;
+      },
+    },
+    {
+      title: SessionTableTitle.memUsage,
+      className: cx("cl-table__col-session-mem-usage"),
+      cell: session => BytesWithPrecision(session.session.alloc_bytes?.toNumber(), 0) +
+        "/" + BytesWithPrecision(session.session.max_alloc_bytes?.toNumber(), 0),
+      sort: session => session.session.alloc_bytes?.toNumber(),
+    },
+    {
+      title: SessionTableTitle.statement,
+      className: cx("cl-table__col-query-text"),
+      cell: (session) => {
+        if (!(session.session.active_queries?.length > 0)) {
+          return "N/A";
+        }
+        const stmt = session.session.active_queries[0].sql;
+        const anonStmt = session.session.active_queries[0].sql_anon;
+        return <StatementLink
+          statement={ stmt }
+          anonStatement={ anonStmt }
+          implicitTxn={ session.session.active_txn?.implicit }
+          search={ "" }
+          app={ "" }
+        />;
+      },
+    },
+    {
+      title: SessionTableTitle.actions,
+      className: cx("cl-table__col-session-actions"),
+      cell: ({session}) => {
+        const sessionId = session.id.join("");
+        const menuItems = (
+          <Menu>
+            <Menu.Item
+              disabled={session.active_queries?.length === 0}
+            >
+              <Anchor
+                onClick={() => {
+                  if (session.active_queries?.length > 0) {
+                    terminateQueryRef?.current?.showModalFor({
+                      query_id: session.active_queries[0].id,
+                      node_id: session.node_id.toString(),
+                      username: session.username,
```

Comment:
This is confusing but the username here should actually be the user who is canceling the session, not the user whose session it is. (I believe this field exists because when this gets run as a SQL command, the request is always made as root.) The way you have it right now, a user with CANCELQUERY privileges can't cancel other users' sessions.